sub/sd_lavc.c

```diff
@@ -105,24 +105,19 @@ static int init(struct sd *sd)
         int64_t format;
```

Comment:
there are two `goto error;` above here so correct error behavior for probing is incomplete

Reply:
You mean this one?
https://github.com/mpv-player/mpv/blob/d61d2946653a33d961ef029b6f73c3a20b26f9eb/sub/dec_sub.c#L174-L175 

I guess that works, indeed.